src/JustSaying/Messaging/Middleware/ExactlyOnce/HandlerMiddlewareBuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using JustSaying.Messaging.MessageHandling;
+using JustSaying.Messaging.Middleware.Handle;
+using Microsoft.Extensions.Logging;
+using HandleMessageMiddleware = JustSaying.Messaging.Middleware.MiddlewareBase<JustSaying.Messaging.Middleware.Handle.HandleMessageContext, bool>;
+
+namespace JustSaying.Messaging.Middleware.ExactlyOnce
```

Comment:
This API would be more discoverable if it was in the same namespace as the builder it's extending.

Reply:
Done 👍